metastore.go

```diff
@@ -240,6 +238,24 @@ func (tx *metatx) createSeries(database, name string, tags map[string]string) (*
 	return s, nil
 }
 
+func (tx *metatx) dropSeries(database, name string, seriesID uint32) error {
+	measurmentBucket := tx.Bucket([]byte("Databases")).Bucket([]byte(database)).Bucket([]byte("Series")).Bucket([]byte(name))
+
+	c := measurmentBucket.Cursor()
+
+	for k, _ := c.First(); k != nil; k, _ = c.Next() {
```

Comment:
And actually, an even better way to go would be to pass this thing a collection of structs:

```go
type seriesDrop struct {
  measurmentName string
  ids []uint316
```

Then you pass that array to the drop series method and you can remove everything in a single tx.